builder.config.json

```diff
@@ -347,7 +347,21 @@
         "id": "14",
         "name": "favourite_fruits",
         "display_label": "Favourite fruits (multiple_choice)",
-        "data_type": "multiple_choice"
+        "data_type": "multiple_choice",
+        "choices": [
```

Comment:
is this the schema we get from tree-builder ?

Reply:
It seems so 🙂

<img width="599" alt="Screen Shot 2022-08-05 at 5 14 01 PM" src="https://user-images.githubusercontent.com/18130716/183107927-31cec0d5-54f5-4006-85fe-6e563304040a.png">